extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1737,9 +1737,10 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   renderOpenAttachmentUI_() {
-    const attachmentEl = this.element.querySelector(
-      'amp-story-page-attachment'
-    );
+    // AttachmentEl can be either amp-story-page-attachment or amp-story-page-outlink
+    const attachmentEl = this.element.querySelectorAll(
+      'amp-story-page-attachment, amp-story-page-outlink'
+    )[0];
```

Comment:
You can actually keep the `querySelector()` and remove the `[0]`, that'll do the same thing